public/app/plugins/datasource/prometheus/querybuilder/shared/OperationEditor.tsx

```diff
@@ -245,6 +245,9 @@ function callParamChangedThenOnChange(
 
 const getStyles = (theme: GrafanaTheme2) => {
   return {
+    cardWrapper: css({
```

Comment:
last question, why are we adding `cardWrapper` and `height: '100%'`. I tested UI without these changes and I don't see any difference. So I am probably missing something, just curious.

Reply:
this actually fixes a bug that i realised i added in a previous PR.
ideally i should have made a separate pr for this but i found it whilst working on this so i decided to fix it here.

**with**
<img width="745" alt="Screenshot 2023-03-14 at 09 58 33" src="https://user-images.githubusercontent.com/34524710/224965483-895e4b41-ccb9-44cf-aebe-45a9587a927f.png">


**without**
<img width="746" alt="Screenshot 2023-03-14 at 09 58 57" src="https://user-images.githubusercontent.com/34524710/224965437-0535746a-e610-4e81-bb95-3e82ddfde0a8.png">